src/main/java/games/strategy/triplea/ui/history/HistoryLog.java

```diff
@@ -186,6 +185,7 @@ public void printRemainingTurn(final HistoryNode printNode, final boolean verbos
     }
     final List<String> moveList = new ArrayList<>();
     boolean moving = false;
+    DefaultMutableTreeNode curNode = printNode;
```

Comment:
`printNode` is `final`, so there is no chance that it can be modified somewhere between old line 161 and new line 188.  Therefore, moving the declaration of `curNode` to this line should result in the same initial value.